### PR TITLE
feat: get rid of default maintainer history tables

### PR DESCRIPTION
**Pull request by MiguelAHM**

### CODE REVIEW

File: whois-commons/src/main/resources/patch/default-maintainer-history.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DROP TABLE IF EXISTS `default_maintainer_history`;
```
Comment (eshryane) [inline whois-commons/src/main/resources/patch/default-maintainer-history.sql:1]: I think we need to version patch files against the current or next version of Whois, also update the `version` table to match - check the other patch files for examples

File: whois-commons/src/main/resources/patch/whois-1.106.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+-- delete default maintainer history tables. Not longer in use.
+
+DROP TABLE IF EXISTS `default_maintainer_history`;
+DROP TABLE IF EXISTS `default_maintainer_sync_history`;
+
```
Comment (maggarwal13) [inline whois-commons/src/main/resources/patch/whois-1.106.sql:5]: It should be Internals patch file?

Comment (maggarwal13) [reply whois-commons/src/main/resources/patch/whois-1.106.sql:5]: check internals-1.102.sql as an example

Comment (MiguelAHM) [reply whois-commons/src/main/resources/patch/whois-1.106.sql:5]: what is the difference here? how could I know that this is an internals?

Comment (maggarwal13) [reply whois-commons/src/main/resources/patch/whois-1.106.sql]: By which schema table exists in.  If tables exists in whois schema it should be in whois patch file, if table exists in internals schema it should be in internals schema

Comment (MiguelAHM) [reply whois-commons/src/main/resources/patch/whois-1.106.sql]: ok, makes sense

**Review by maggarwal13 — APPROVED** [review]

**Pull request merged**